database/migrations/2023_09_19_094751_create_wiki_lifecycle_events_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateWikiLifecycleEventsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('wiki_lifecycle_events', function (Blueprint $table) {
+            $table->id();
+            $table->timestamps();
+            $table->timestampTz('first_edited')->nullable();
```

Comment:
If I understand it correctly, all Wikis on wikibase.cloud run on UTC, so we could also drop the Timezone information here as it's always `+00:00`. Then again it's probably safer to already add it.